requirements.txt

```diff
@@ -4,3 +4,4 @@ mysqlclient
 pymssql<=2.2.5
 sqlalchemy
 sqlalchemy-hana
+vobject
```

Comment:
I actually don't mind very much where to put this, if other reviewers feel it should go there, I'll be happy to oblige.

Reply:
`vobject` also offers `vtodo` if you want to support it (based on `mail.activity`) which works quite nicely if you use a html2text function for the description. This would open a new possibility and would go further than a way to sync calendar clients. This would mean that it shouldn't be in calendar if we make it possible.

Reply:
In my opinion it would be slightly easier to discover this module (by functional users) and be more intuitive in /OCA/calendar even though if it would support vtodo in the future.